app/http.php

```diff
@@ -1103,21 +1163,21 @@ function (string $runtimeId, ?string $payload, string $path, string $method, arr
             $endTime = \microtime(true);
             $duration = $endTime - $startTime;
 
-            $header['x-open-runtimes-encoding'] = 'original';
+            $header['x-open-runtimes-body-encoding'] = 'original';
             $execution = [
                 'statusCode' => $statusCode,
                 'headers' => $headers,
                 'body' => $body,
-                'logs' => \mb_strcut($logs, 0, 1000000), // Limit to 1MB
-                'errors' => \mb_strcut($errors, 0, 1000000), // Limit to 1MB
+                'logs' => \mb_strcut($logs, 0, 5000000), // Limit to 5MB
+                'errors' => \mb_strcut($errors, 0, 5000000), // Limit to 5MB
```

Comment:
This will remove our message we add after 5MB. Limit here shouldn't be needed. It was probably here for V2. Let's do this strcut in V2 function instead. Then we don't need to do any cutting here.